tensorboard/webapp/core/effects/core_effects_test.ts

```diff
@@ -153,14 +167,19 @@ describe('core_effects', () => {
           .expectOne('data/plugins_listing')
           .error(new ErrorEvent('FakeError'), {status: 404});
 
+        fetchPolymerRunsSubjects[0].next([{id: '1', name: 'Run 1'}]);
+        fetchPolymerRunsSubjects[0].complete();
+
         expect(recordedActions).toEqual([
           coreActions.pluginsListingRequested(),
           coreActions.environmentLoaded({
             environment: createEnvironment(),
           }),
+          coreActions.polymerRunsFetchRequested(),
```

Comment:
Should we have a similar test but where polymerRunsFetchFailed and pluginsListingFetchSucceeded?